### DISCUSSION TITLE
Where to Store Logo Within Application

### DISCUSSION THREAD

**Discussion by shawnweeks** [Budibase/budibase#10261] [Help] [answered]
Currently the page logo is just a link to an external url. Where do I store that logo within my application if I don't have external hosting for the logo?

**Comment by andz-bb** [comment, marked as answer]
Hey @shawnweeks 

There's no quick way to select one of your files as a page logo but a workaround for this would be to create a separate Budibase DB table with an Attachments column. Then if you add your logo as a row to this table that image will be available on its own URL. If you right click on the image thumbnail in the data section table view and open it in a new tab, the image URL will be available to copy from the address bar and paste into your Logo field.

<img width="963" alt="opening the image in the new tab" src="https://user-images.githubusercontent.com/110921612/231416096-1db163c5-4221-4545-ab95-69b8115b3373.png">

<img width="966" alt="getting the image url" src="https://user-images.githubusercontent.com/110921612/231416127-ed440f0b-8232-44e4-bab5-27c9322f0316.png">

<img width="576" alt="pasting url into logo field" src="https://user-images.githubusercontent.com/110921612/231416159-80c3a5eb-f5c1-4f4f-9a74-d4b4f46128a3.png">

Hope this helps! 😊

**Reply by NicolasBio** [reply]
I don't think this works anymore? The link expires after a certain time, or am i missing something?

**Reply by ConorWebb96** [reply]
This is a bindable field now. You will still can do this. However, you'll just have to include a data provider that has the image's object. From here you can use a binding to get the url to put inside the logo area now.
<img width="487" alt="Screenshot 2024-07-11 at 11 54 38" src="https://github.com/Budibase/budibase/assets/126772285/c2092b26-f87b-4dfe-9630-7e63bb4d9b9e">

<img width="1512" alt="Screenshot 2024-07-11 at 11 55 00" src="https://github.com/Budibase/budibase/assets/126772285/4e9e16a3-3fd8-4eb1-a50d-3725ee84af84">

<img width="1511" alt="Screenshot 2024-07-11 at 11 54 54" src="https://github.com/Budibase/budibase/assets/126772285/47c5b83c-fda4-4177-898d-49f1a51e5de5">

Alternatively you can just use the organisation's branding in the branding area and it will change the logo across all applications.
<img width="1155" alt="Screenshot 2024-07-11 at 11 50 13" src="https://github.com/Budibase/budibase/assets/126772285/74b0fc41-edd8-4b73-ab13-369290771b98">

**Reply by NicolasBio** [reply]
Thanks for writing this up!  I didn't think of this solution.

**Comment by lburtwistle** [comment]
Also, if you are wanting to use the same logo you uploaded in your organization settings, a quick way to do this is to use this uri path -  /global/settings/logoUrl
